package.json

```diff
@@ -63,13 +63,13 @@
     "vinyl-fs": "~2.4.3"
   },
   "devDependencies": {
-    "@types/chalk": "0.4.31",
```

Comment:
It looks like you accidentally reverted 00a04d10eac2b879f410809ce8ef4dc2f1a4d635 in a merge, could you undo that?